src/enums/index.ts

```diff
@@ -20,3 +22,4 @@ export * from './warningTypes'
 
 // external enums
 export { CorpTypeCd, CorpClass } from '@bcrs-shared-components/corp-type-module'
+export { AccountTypes } from '@bcrs-shared-components/enums'
```

Comment:
This enum should be obsolete now, right?

Reply:
Oh, still used for now, same as in Edit UI:

![image](https://github.com/user-attachments/assets/d55fd45a-4da9-48f5-b049-73f72a28e05e)

Reply:
27536 is implemented. You can now update this code (and get rid of AccountTypes).

Reply:
I see isSbcFieldOfficeStaff is updated and the subject enum is still used to type a property in an interface. Looks like nothing more is needed. Resolving.